SU-SWS/saa_victoria_subtheme
# ADAPT-662: Fixed icon color override issue with moodule update.

# READY FOR REVIEW
# Summary
- Fixes icon color in secondary button

# Review By (Date)
- 9/4

# Urgency
- low - not an issue until d8core release

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. On an updated cardinalsites environment, a module update messes up external link styles for secondary button. Make sure secondary buttons with icons all work with the correct colors.

test code:
```
<p>Here are the 3 classic button types:</p>

<p><a class="su-button" href="&lt;front&gt;">Button</a></p>

<p><a class="su-button--secondary" href="&lt;front&gt;">Secondary Button</a></p>

<p><a class="su-button--big" href="&lt;front&gt;">Big Button</a></p>

<p>All three of these button types support adding an icon. It can look something like this:<br />
&nbsp;</p>

<p><a class="su-button su-link--download" href="&lt;front&gt;">Download Button</a></p>

<p><a class="su-button su-link--external" href="&lt;front&gt;">External Button</a></p>

<p><a class="su-button su-link--internal" href="&lt;front&gt;">Internal Button</a></p>

<p><a class="su-button su-link--action" href="&lt;front&gt;">Button</a></p>

<p><a class="su-button su-link--video" href="&lt;front&gt;">Button</a></p>

<p><a class="su-button--secondary su-link--download" href="&lt;front&gt;">Download Button</a></p>

<p><a class="su-button--secondary su-link--external" href="&lt;front&gt;">External Button</a></p>

<p><a class="su-button--secondary su-link--internal" href="&lt;front&gt;">Internal Button</a></p>

<p><a class="su-button--secondary su-link--video" href="&lt;front&gt;">Internal Button</a></p>

<p><a class="su-button--big su-link--action" href="&lt;front&gt;">Big Action Button</a></p>

<h3>Links</h3>

<p>Here are the link types:</p>

<p><a class="su-link" href="&lt;front&gt;">Button</a></p>

<p><a class="su-link su-link--download" href="&lt;front&gt;">Download Button</a></p>

<p><a class="su-link su-link--external" href="&lt;front&gt;">External Button</a></p>

<p><a class="su-link su-link--internal" href="&lt;front&gt;">Internal Button</a></p>

<p><a class="su-link su-link--video" href="&lt;front&gt;">Internal Button</a></p>

<p><a class="su-link su-link--action" href="&lt;front&gt;">Button</a></p>

```

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
